enterprise_access/apps/api/serializers/subsidy_access_policy.py

```diff
@@ -239,6 +239,81 @@ class SubsidyAccessPolicyDeleteRequestSerializer(serializers.Serializer):
     )
 
 
+class SubsidyAccessPolicyUpdateRequestSerializer(serializers.ModelSerializer):
+    """
+    Request Serializer for PUT or PATCH requests to update a subsidy access policy.
+
+    For views: SubsidyAccessPolicyViewSet.update and SubsidyAccessPolicyViewSet.partial_update.
+    """
+    class Meta:
+        model = SubsidyAccessPolicy
+        fields = (
+            'description',
+            'active',
+            'catalog_uuid',
+            'subsidy_uuid',
+            'access_method',
+            'spend_limit',
+            'per_learner_spend_limit',
+            'per_learner_enrollment_limit',
+        )
+        extra_kwargs = {
+            'description': {
+                'required': False,
+                'allow_blank': False,
+                'min_length': None,
+                'max_length': 200,
+                'trim_whitespace': True,
+            },
+            'active': {
+                'allow_null': False,
+                'required': False,
+            },
+            'catalog_uuid': {
+                'allow_null': False,
+                'required': False,
+            },
+            'subsidy_uuid': {
+                'allow_null': False,
+                'required': False,
+            },
+            'access_method': {
+                'allow_null': False,
+                'required': False,
+            },
+            'spend_limit': {
+                'allow_null': True,
+                'required': False,
+            },
+            'per_learner_enrollment_limit': {
+                'allow_null': True,
+                'required': False,
+            },
+            'per_learner_spend_limit': {
+                'allow_null': True,
+                'required': False,
+            },
+        }
+
+    def validate(self, attrs):
+        """
+        Raises a ValidationError if any field not explicitly declared
+        as a field in this serializer defintion is provided as input.
+        """
+        unknown = sorted(set(self.initial_data) - set(self.fields))
+        if unknown:
+            raise serializers.ValidationError("Field(s) are not updatable: {}".format(", ".join(unknown)))
+        return attrs
```

Comment:
Does this need to copy/inherit any logic to check that "no more" custom fields are provided than allowed for the policy type given?

https://github.com/openedx/enterprise-access/blob/888b77d40fd666b39c85a7283a31b4f7c78b6c02/enterprise_access/apps/api/serializers/subsidy_access_policy.py#L141-L146